assets/.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
```

Comment:
I added this file, because the bat files in our assets had just LF endings, and when I was running `state activate ActiveState/Perl-5.32 --default` as an Administrator the activation script was not interpreted correctly.  But *only* as an Administrator user.  It worked otherwise.  As debugging the issues also meant fixing it, I decided to put it into this PR. Let me know if you want me to put it in a separate bug instead...